code/datums/uplink_item.dm

```diff
@@ -132,15 +132,15 @@ var/list/uplink_items = list()
 	name = "Armored Labcoat"
 	desc = "An armored labcoat with the ability to change how it looks into any standard Nanotrasen issue labcoat."
 	item = /obj/item/clothing/suit/labcoat/chameleon
-	cost = 4
+	cost = 2
 	jobs = list("Chemist", "Medical Doctor", "Chief Medical Officer", "Geneticist", "Virologist", "Research Director", "Scientist", "Roboticist") //all the labcoat jobs
 
 //MEDICAL
 /datum/uplink_item/job_specific/voodoo
 	name = "Voodoo Doll"
 	desc = "An artifact provided by the Wizard Federation, it allows you to attach any item to it and manipulate anyone who has ever touched said item, be it from moving them around, making them say stuff and stabbing them with objects. The Wizard Federation has notified us that the victim will be made alert of the user's general location if used too aggresively."
 	item = /obj/item/weapon/storage/box/syndie_kit/voodoo
-	cost = 10
+	cost = 2
```

Comment:
Fair enough. I don't think it's used particularly often, but I'll boost it from that at least